[ci] Fix arm CI job by running setup.sh

larryliu0820 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #1945

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D53681569](https://our.internmc.facebook.com/intern/diff/D53681569)

larryliu0820 committed:
[ci] Fix arm CI job by running setup.sh

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

larryliu0820 commented:
@larryliu0820 has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D53681569).

dbort commented on .github/workflows/trunk.yml:
```diff
@@ -153,6 +153,11 @@ jobs:
+
+        git config --global user.email "github_executorch@arm.com"
+        git config --global user.name "Github Executorch"
+        bash examples/arm/setup.sh --i-agree-to-the-contained-eula
```
Based on the readme file, this might need to be
```
bash examples/arm/setup.sh --i-agree-to-the-contained-eula /opt/arm-sdk
```
to match the arg that's passed to run.sh on line 163.

Or maybe that call shouldn't have `/opt/arm-sdk` in it -- it's weird to have a root-owned directory like that used as a "scratch dir"

mergennachin commented:
Looks like it hasn't fixed it

https://github.com/pytorch/executorch/actions/runs/7878561349/job/21497079824?pr=1945&fbclid=IwAR3tRCoJLG1LD5BSUtDcJkWBbhxzJWAHy6I8XQwbNkaJBU5StPuLowr9XaY

@robell , @Jerry-Ge

We will disable this CI job for the time being and created this PR (https://github.com/pytorch/executorch/issues/1947)

We are currently trying make all CI jobs green and this is the last one (https://hud.pytorch.org/hud/pytorch/executorch/main/1?per_page=50)

larryliu0820 committed:
Update on "[ci] Fix arm CI job by running setup.sh"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D53681569](https://our.internmc.facebook.com/intern/diff/D53681569)

[ghstack-poisoned]

Jerry-Ge commented:
@robell 's this patch should be able to fix that: https://github.com/pytorch/executorch/pull/1909

larryliu0820 commented:
doesn't work
